docs/topics/spiders.rst

```diff
@@ -289,6 +289,31 @@ Spiders receive arguments in their constructors::
             self.start_urls = ['http://www.example.com/categories/%s' % category]
             # ...
 
+To pass more complex arguments like dicts and lists use ``arg-json`` instead.
+Each argument must be JSON-encoded. For example::
+
+    scrapy crawl myspider --arg-json start_urls='["http://www.example.com", "http://dmoz.com"]'
+
+Spider will now start crawling with urls passed via command line.
+
+Note:: JSON args that contain whitespaces or characters interpreted by Bash like
```

Comment:
Sphinx provides another syntax for notes, see http://sphinx-doc.org/markup/para.html